packages/phishing-controller/src/utils.ts

```diff
@@ -1,3 +1,6 @@
+import { sha256 } from 'ethereum-cryptography/sha256';
+import { toHex } from 'ethereum-cryptography/utils';
```

Comment:
Instead of ethereum-cryptography we usually use `@metamask/controller-utils` for the `toHex`.

Reply:
Looks like the `toHex` function from `@metamask/controller-utils` must follow these guidelines "Strictly check if a string is a valid hex string. A valid hex string must start with the "0x"-prefix." which does not fix our use case. We can either modify the helper function or pull in this package `ethereum-cryptography` which we also use in `@metamask/accounts-controller`

Reply:
Ok so, `toHex` from `ethereum-cryptography/utils` is actually the [binaryToHex](https://github.com/ethereum/js-ethereum-cryptography/blob/master/src/utils.ts#L8) from `@noble/hashes/utils` - [implementation here](https://github.com/paulmillr/noble-hashes/blob/main/src/utils.ts#L63). Can we keep the `binaryToHex` naming (and maybe use the `@noble/hashes/utils` that is also already used by the notifications controller). This will ensure a clear distinction from the `toHex` from `@metamask/controller-utils`